packs/travis_ci/actions/get_git_token.py

```diff
@@ -0,0 +1,23 @@
+from lib.action import TravisCI
+import json
+import requests
+
+
+class GetGitToken(TravisCI):
+    def run(self):
+        username = self.config['username']
+        password = self.config['password']
+        url = 'https://api.github.com/authorizations'
+        host = 'api.github.com'
+        ContentType = 'application/json'
+        data = {"scopes": ["read:org", "user:email",
+                "repo_deployment", "repo:status", "write:repo_hook"],
+                "note": "token for travis api"}
+        headers = {}
+        headers['Host'] = host
```

Comment:
Yes, we need to automate the git token procedure as much as we can. First, the user has to authorize the app to grant access, after that `we need the git token in order to get Travis CI token`.  Getting Git token was also manual and user interaction was needed. So, we wrote the action to automate that as well and the user interaction should be reduced to the minimal.